gateway-server/src/main/java/org/apache/knox/gateway/services/token/impl/TokenStateDatabase.java

```diff
@@ -28,7 +28,7 @@
 
 public class TokenStateDatabase {
   static final String TOKENS_TABLE_NAME = "KNOX_TOKENS";
-  private static final String ADD_TOKEN_SQL = "INSERT INTO " + TOKENS_TABLE_NAME + "(token_id, issue_time, expiration, max_lifetime) VALUES(?, ?, ?, ?)";
+  private static final String ADD_TOKEN_SQL = "INSERT INTO " + TOKENS_TABLE_NAME + "(id, token_id, issue_time, expiration, max_lifetime) VALUES(?, ?, ?, ?, ?)";
```

Comment:
I was thinking that we would leave this as token.id and add a passcode col that was a hash.